src/lib/supabase/library/library.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  mockGetAllLibraryItems,
+  mockSearchLibraryItemsByAvailability,
+  mockSearchLibraryItemsByCategory,
+  mockSearchLibraryItemsByName,
+  mockSearchLibraryItemsByTags,
+  mockSearchAllLibraryItems,
+} from './mockData';
+import {TLibraryItems} from '~/models/models';
+
+//Stub Cloud Functions for the Library Page
+
+const getAllLibraryItems = async (): Promise<ReadonlyArray<TLibraryItems>> => {
+  const response = await new Promise<Array<TLibraryItems>>(resolve => {
+    resolve(mockGetAllLibraryItems);
+  });
+  return response;
+};
+
+const searchLibraryItemsByName = async (
+  name: string
```

Comment:
Let's get these warnings sorted by either commented out these methods, figuring out how we're utilizing these params, or by console logging them or something. 

eg: `'name' is defined but never used`